backend/pkg/api/connect/service/acl/service.go

```diff
@@ -156,10 +156,52 @@ func (s *Service) CreateACL(ctx context.Context, req *connect.Request[v1alpha1.C
 }
 
 // DeleteACLs implements the handler for the delete ACL endpoint.
-func (*Service) DeleteACLs(context.Context, *connect.Request[v1alpha1.DeleteACLsRequest]) (*connect.Response[v1alpha1.DeleteACLsResponse], error) {
-	return nil, apierrors.NewConnectError(
-		connect.CodeUnimplemented,
-		errors.New("endpoint is not implemented"),
-		apierrors.NewErrorInfo(v1alpha1.Reason_REASON_CONSOLE_ERROR.String()),
-	)
+func (s *Service) DeleteACLs(ctx context.Context, req *connect.Request[v1alpha1.DeleteACLsRequest]) (*connect.Response[v1alpha1.DeleteACLsResponse], error) {
+	// TODO: Ensure that neither req, req.Msg or req.Msg.Filter can never be nil
+	kafkaReq, err := s.kafkaClientMapper.deleteACLFilterToDeleteACLKafka(req.Msg.Filter)
+	if err != nil {
+		return nil, apierrors.NewConnectError(
+			connect.CodeInternal, // Internal because all input should already be validated, and thus no err possible
+			err,
+			apierrors.NewErrorInfo(v1alpha1.Reason_REASON_CONSOLE_ERROR.String()),
+		)
+	}
+
+	res, err := s.consoleSvc.DeleteACLsKafka(ctx, kafkaReq)
+	if err != nil {
+		return nil, apierrors.NewConnectError(
+			connect.CodeInternal,
+			err,
+			apierrors.NewErrorInfo(v1alpha1.Reason_REASON_KAFKA_API_ERROR.String(), apierrors.KeyValsFromKafkaError(err)...),
+		)
+	}
+
+	if len(res.Results) != 1 {
+		// Should never happen since we only create one ACL, but if it happens we want to err early.
```

Comment:
nit: Maybe here you meant 
```suggestion
		// Should never happen since we only delete one ACL, but if it happens we want to err early.
```